### ISSUE TITLE
Formatting is strange

### ISSUE THREAD

**Issue by xCONFLiCTiONx** [samwilson/embed-wikimedia#18]
I think my WordPress site is having issues with your plugin as the name is on the left of the image as a link.

It should be the image on the left without that link with info at the bottom.

This is how it currently looks:
![image](https://user-images.githubusercontent.com/12835195/74768398-1f0b9180-524e-11ea-989b-86539eebd877.png)

This is how it LIKELY should look:
![image](https://user-images.githubusercontent.com/12835195/74768312-f4b9d400-524d-11ea-9d6b-c208c821ffd7.png)

Thank you!

**Comment by samwilson** [comment]
This might be a theme styling issue. I agree that it looks odd when the image also contains the same text as the title, but often it's different (e.g. a photo).

Totally agree that the default styling should be better though. Do you think an `h3` above the image would be better for the article title? At the moment it's constructed like this:

```php
$out = '<blockquote class="embed-wikimedia">'
	. '<a href="' . $url . '"><strong>' . $info['displaytitle'] . '</strong></a>'
	. $img
	. $info['extract_html']
	. '</blockquote>';
```

**Comment by xCONFLiCTiONx** [comment]
Is it at all possible to give an option on what's displayed? For example
the title is redundant in my case because it's used in a post. Only the
image and summary are needed.

We really were looking for something similar to the [IMDb shortcode](https://nlec.tv/wp-admin/plugin-install.php?tab=plugin-information&plugin=shortcode-imdb&TB_iframe=true&width=600&height=550) [DEMO](http://demo.pluginpress.net/shortcode-imdb/2018/11/29/shortcode-imdb/) plugin
but for wiki and yours is the closest we've found.

Thank you for replying.

**Comment by samwilson** [comment]
No, the way embeds work they are not able to take any parameters. These days, however, everything's done through blocks, so I think the better solution here (for customisability) is to make a new Wikimedia blocks (there already are three in this plugin, but replicate the embed functionality). Then the new blocks can take whatever parameters (such as image yes/no, title yes/no, custom caption for Commons, etc.).

That's more work than I can do at the moment, though! Sorry.
